object-manipulation-2/src/invert.js

```diff
@@ -0,0 +1,27 @@
+/* exported invert */
+/*
+convert the object to array using Object.entries method
+swap the each array index
+convert the resulting array to object using Object.fromEntries method
+*/
+// function invert(source) {
+//   var newArray = Object.entries(source);
+//   for (var i = 0; i < newArray.length; i++) {
+//     var temp = newArray[i][0];
+//     newArray[i][0] = newArray[i][1];
+//     newArray[i][1] = temp;
+//   }
+//   var newObject = Object.fromEntries(newArray);
+//   return newObject;
+// }
```

Comment:
Please remove commented code before submitting work for review.

Reply:
This code is an alternative solution for the problem, may I keep it for future reference?